hyperspy/io_plugins/digital_micrograph.py

```diff
@@ -858,6 +858,25 @@ def _get_EELS_exposure_time(self, tags):
         else:
             _logger.info("EELS exposure time can't be read.")
 
+    def _get_CL_detector_type(self, tags):
+        if "Acquisition_Mode" in tags and tags["Acquisition_Mode"] == "Parallel dispersive":
+            return "CCD"
+        elif "Acquisition_Mode" in tags and tags["Acquisition_Mode"] == "Serial dispersive":
+            return "PMT"
+        else:
+            _logger.info("CL detector type can't be read.")
+
+    def _get_CL_exposure_time(self, tags):
```

Comment:
Make it more generic and merge it with `_get_EELS_exposure_time`? https://github.com/hyperspy/hyperspy/blob/d42c03d5986c649d2121fa34cfe361e9506e5a86/hyperspy/io_plugins/digital_micrograph.py#L847-L859